source/jsonnet/northern-ireland/household/blocks/accommodation/renewables_energy_systems.jsonnet

```diff
@@ -3,9 +3,9 @@ local rules = import 'rules.libsonnet';
 
 {
   type: 'Question',
-  id: 'renewables',
+  id: 'renewables-energy-systems',
```

Comment:
```suggestion
  id: 'renewable-energy-systems',
```

Reply:
Cracking spot that, it took me a couple of minutes to even spot what the difference was